recipes/eip712/license.txt

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
The license is already included in the source tarball so you don't need to (and shouldn't) include it in the recipe. Please go ahead and remove.

Reply:
removed.

Reply:
@dopplershift I removed the license.txt file, then the tests failed. So I removed the reference again and it still fails. Sorry to be a nuisance, but I don't understand if I should or not add this file and the reference in meta.yaml...